core/forms.py

```diff
@@ -135,6 +135,7 @@ def hydrated_fielsets(self):
 
 class TaggableModelForm(forms.ModelForm):
     tags = TagField(
+        label=_("Tags"),

```

Comment:
When overriding fields in Django forms we need to explicitly set field attributes

Reply:
Thanks, good catch!